ecommerce_extensions/payment/processors/edupay.py

```diff
@@ -261,6 +281,35 @@ class of 'seat'.  Return None if no such products were found.
 
         return None
 
+    @staticmethod
+    def get_seat_list(basket):
+        """
+        Return the products encountered in the basket with the product
+        class of 'seat'.  Return None if no such products were found.
+
+        Arguments:
+            basket (Basket): Basket being purchased via the payment processor.
+
+        Returns:
+            product_list: All the products in the basket with the class seat.
+        """
+        product_list = []
+        try:
+            seat_class = ProductClass.objects.get(slug='seat')
+        except ProductClass.DoesNotExist:
+            # this occurs in test configurations where the seat product class is not in use
+            return None
+
+        for line in basket.lines.all():
```

Comment:
```suggestion
        product_list = [line.product for line in basket.lines.all() if isinstance(line.product, seat_class)]
```

that works ?

Reply:
@andrey-canon that line throws an error, TypeError: isinstance() arg 2 must be a type or tuple of types